### PR TITLE
Update Node 22

### PR DESCRIPTION
**Pull request by johnwatson484**
# Description

Please include a summary of the changes i.e. Node.js version updates, libraries patched, or issues added to the policy file.

# Checklist:

- [x] I have ensured the Defra version in the **JOB.env** file matches that in the **Dockerfile**
- [x] I have ensured the Node.js versions in the **image-matrix.json** match the **Dockerfile** and the table in the **README.md**
- [x] I have added newly ignored vulnerabilities to the **POLICY_CONFIGURATION.md**
- [x] I have checked if previously identified vulnerabilities have been patched, and can be removed from the **.grype.yaml** and **POLICY_CONFIGURATION.md** files

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Set default values for build arguments
-ARG DEFRA_VERSION=2.4.0
-ARG BASE_VERSION=22.9.0-alpine3.20
+ARG DEFRA_VERSION=2.4.1
```
Comment (paulsimonandrews) [inline Dockerfile:2]: Not sure if we have a properly defined process for bumping the Defra version, but as this is a "biggish" change maybe we should go to 2.5.0?

File: JOB.env [hunk 2]
```diff
@@ -1,2 +1,2 @@
-DEFRA_VERSION=2.4.0
+DEFRA_VERSION=2.4.1
```
Comment (paulsimonandrews) [inline JOB.env:1]: Bump this to 2.5.0 too if we change the version in Dockerfile to that.

**Review by paulsimonandrews — COMMENTED** [review]
Go to version 2.5.0 instead of 2.4.1 as changing the `latest` tag might have biggish changes to some consumer?

File: Dockerfile (hunk 1 above)
Comment (johnwatson484) [reply Dockerfile]: I think you're being generous with this comment as the README clearly states I've got this wrong!

I looked through the history and saw it was inconsistent, so just opted for patch.  However, the README says that updating Node or Alpine should be a minor update.

I have corrected it. :)

**Review by paulsimonandrews — APPROVED** [review]
LGTM

**Pull request merged**